client/src/components/MainPage/TranslationsPage/index.jsx

```diff
@@ -100,12 +100,13 @@ class TranslationsPage extends Component {
     if (!question && !error) {
       return <Loading />;
     }
+    const questionText = question.question;
```

Comment:
@ali-7 Thanks for using a meaningful variable name here instead of just using 'question.question' everywhere.  That'll help others who work in this code later.